ios/INVRegionMonitor.m

```diff
@@ -138,13 +138,19 @@ - (BOOL)_isRegionMonitoringPossible:(CLAuthorizationStatus)status {
     );
 }
 
-- (void)_sendRegionChangeEventWithIdentifier:(NSString *)identifier didEnter:(BOOL)didEnter didExit:(BOOL)didExit {
+- (void)_sendRegionChangeEventWithIdentifier:(NSString *)identifier didEnter:(BOOL)didEnter didExit:(BOOL)didExit locationManager:(CLLocationManager *)locationManager {
     NSDictionary *body = @{
         @"region": @{
             @"identifier": identifier,
         },
         @"didEnter": @(didEnter),
         @"didExit": @(didExit),
+        @"accuracy": @(locationManager.location != nil ? locationManager.location.horizontalAccuracy : 99999),
+        @"bearing": @(locationManager.location != nil ? locationManager.location.course : 0),
+        @"location": @{
```

Comment:
If the location is nil, what will the location be in JavaScript-land? I think it should be `null` instead of `{ latitude: 0, longitude: 0 }`. Do you agree?